user_input.py

```diff
@@ -0,0 +1,59 @@
+from textwrap import dedent
+from time import sleep
+from random import choice
```

Comment:
```suggestion
from time import sleep
```

Removes unused imports